main.js

```diff
@@ -12,9 +12,42 @@ window.addEventListener('load', trigger.bind(null, 'load'));
 window.addEventListener('load', trigger.bind(null, 'DOMContentLoaded'));
 document.addEventListener('DOMContentLoaded', trigger.bind(null, 'DOMContentLoaded'));
 
+document.onreadystatechange = function () {
+	if (document.readyState === 'complete') {
+		trigger('DOMContentLoaded');
+		trigger('load');
+	}
+};
+
 if (document.readyState === 'complete') {
-	trigger('load');
-	trigger('DOMContentLoaded');
-} else if (document.readyState === 'interactive') {
```

Comment:
Unfortunately this will suffer the same problem. If the Origami bundle is included in the middle of "body", with this modification it will still fire instantly the o.DOMContentLoaded event, which will trigger 'init' of the modules, and those modules could miss component declarations which are in the body after the origami bundle inclusion.

So basically, there will be no errors, but some components will silently fail to initialize.

And yes, the problem is IE>=9, definitely including IE10 as well.